recipes/libsrtp/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class ConanRecipe(ConanFile):
+    name = "libsrtp"
+    description = (
+        "This package provides an implementation of the Secure Real-time Transport"
+        "Protocol (SRTP), the Universal Security Transform (UST), and a supporting"
+        "cryptographic kernel."
+    )
+    topics = ("conan", "libsrtp", "srtp")
+    homepage = "https://github.com/cisco/libsrtp"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-3-Clause"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "enable_openssl": [True, False],
```

Comment:
We usually name the openssl options, `with_openssl` or `with_tls` or `with_ssl`.
```suggestion
        "with_openssl": [True, False],
```

Reply:
done